fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/scalar/SimplifiedPredicateRule.java

```diff
@@ -343,4 +348,15 @@ private static ScalarOperator ifCall(CallOperator call) {
 
         return ((ConstantOperator) call.getChild(0)).getBoolean() ? call.getChild(1) : call.getChild(2);
     }
+
+    private static ScalarOperator ilikeCall(CallOperator call) {
```

Comment:
could you move the function to `NormalizePredicateRule` ？

Reply:
ok